.github/workflows/linkcheck.yml

```diff
@@ -0,0 +1,36 @@
+name: Link Check
+
+on:
+  schedule:
+    # Run every Monday at 8am to check URL links still resolve
+    - cron: "0 8 * * WED"
+
+jobs:
+  docs:
+    strategy:
+      fail-fast: false
+      matrix:
+        python: ["3.10"]
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Install python version
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python }}
+
+      - name: Install dependencies
+        run: pip install -e .[dev]
+          
+
+      - name: Check links
+        run: tox -e docs -- -b linkcheck
+
+      - name: Keepalive Workflow
+        uses: gautamkrishnar/keepalive-workflow@v1
```

Comment:
Because otherwise it will stop running the scheduled job after 60 days of inactivity: https://github.com/DiamondLightSource/python3-pip-skeleton/issues/104#issuecomment-1400588421

Reply:
Great find!